misc/make-deps.sh

```diff
@@ -56,6 +56,7 @@ if [ -n "$YUM" ]; then
 	$sudo_command $YUM install -y graphviz || true # for debugging
 fi
 if [ -n "$APT" ]; then
+	$sudo_command $APT update -y
```

Comment:
Fantastic! Can you wrap this in a `in_ci github` block, because we already have two other `apt update` runs in this file?

Reply:
We should run an `apt update` here every time regardless of where it runs. This file already suffers far too much from branching. I'll clean up the duplicate `apt update` calls instead